### DISCUSSION TITLE
aux-dependent channel mixing?

### DISCUSSION THREAD

**Discussion by a-marmot** [iNavFlight/inav#7431] [Q&A]
Hi all.  I have a FPV project in which I need to adjust the RC channel mixing on the fly based on an aux input.  The idea is to adjust camera angle with the aux channel, and then remix roll and yaw in accordance with the sin and cos of the camera angle.  I understand that you can change mix weighting in iNav, but it's not obvious how to alter these weightings on the fly based on aux channel input to the FC.

**Comment by DzikuVx** [comment]
Everything you want can be done with Programming Framework with Mixer integration. See https://github.com/iNavFlight/inav/blob/master/docs/Programming%20Framework.md

**Comment by a-marmot** [comment]
Nice, exactly what I was looking for.  Thanks Pawel!
